lte/gateway/c/oai/tasks/mme_app/mme_app_main.c

```diff
@@ -150,6 +151,19 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
               ue_context_p);
           ue_context_p->path_switch_req = false;
         }
+
+        if (ue_context_p->ue_context_rel_cause ==
```

Comment:
a comment here would help. Something on "Check if an offloading request is pending for this UE and part of HA implementation"